kbd-mode.el

```diff
@@ -151,6 +153,193 @@ form `@MACRO-NAME' with `kbd-mode-variable-name-face'."
         (font-lock-add-keywords 'kbd-mode macro-regexp)
       (font-lock-remove-keywords 'kbd-mode macro-regexp))))
 
+;;; Interactive Functions
+
+;;;###autoload
+(defun kbd-mode-start-demo ()
```

Comment:
This doesn't work the first time around, right?  If the hook aren't in place yet, this kills the running kmonad instance but does not create a test instance, leaving one with the default layout in the test buffer

Reply:
You shouldn't be in `demo-buffer` when there is error, I can't reproduce that. It should quit from `demo-buffer`.
```lisp
(defun kbd-mode--show-error ()
  "Show error with compilation buffer."
  (quit-window "*kmonad-demo*")
  (compile (kbd-mode--get-config-validation-command)))
```

Reply:
I don't understand why the code works now, when there is problem in order of operations. Probably in my setup, systemd stopping is faster than kmonad creation.

Reply:
I was talking about coming back to demo-buffer, since I didn't call `kbd-mode-start-demo` every time I've changed config.